### PR TITLE
Bind PN BinaryTrajectories in Python

### PR DESCRIPTION
**Pull request by nilsvu**
## Proposed changes

Useful to test pipeline code like ecc-control.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: tests/Unit/Helpers/PointwiseFunctions/PostNewtonian/Python/Bindings.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+           py::vectorize(&BinaryTrajectories::orbital_frequency),
+           py::arg("time"))
+      .def("angular_velocity",
+           py::vectorize(&BinaryTrajectories::angular_velocity),
+           py::arg("time"))
+      .def("positions", &BinaryTrajectories::positions, py::arg("time"))
+      .def(
+          "positions",
+          [](const BinaryTrajectories& binary_trajectories,
+             const py::array_t<double> time) -> py::array_t<double> {
```
Comment (knelli2) [inline tests/Unit/Helpers/PointwiseFunctions/PostNewtonian/Python/Bindings.cpp]: If you're already going to be doing this in C++, why not just add it to the BinaryTrajectories class.

Comment (nilsvu) [reply tests/Unit/Helpers/PointwiseFunctions/PostNewtonian/Python/Bindings.cpp]: Added commits for this

**Review by knelli2 — CHANGES_REQUESTED** [review]
Code looks good except for the module import errors

**Comment by nilsvu** [comment]
@knelli2 fixed

**Review by knelli2 — APPROVED** [review]

**Comment by knelli2** [comment]
Ignoring unrelated test timeouts

**Pull request merged**